modules/account-lib/src/coin/baseCoin/secp256k1ExtendedKeyPair.ts

```diff
@@ -24,17 +33,25 @@ export abstract class Secp256k1ExtendedKeyPair implements BaseKeyPair {
     this.source = source;
   }
 
+  getPublicKey({ compressed }: { compressed: boolean }): Buffer {
+    return ECPair.fromPublicKey(this.keyPair.publicKey, { compressed }).publicKey;
+  }
+
+  getPrivateKey(): Buffer | undefined {
+    return this.keyPair.privateKey;
+  }
```

Comment:
Add some JS docs explaining the format these keys are returned in.

Reply:
done